101-data-factory-blob-to-sql-copy/azuredeploy.parameters.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dataFactoryName": { "value": "adfsampledf10052016" },
+    "storageAccountName": { "value": "tempazurestorage" },
+    "storageAccountKey": { "value": "iYS5bsP1H08uaw4QY9eRQuPGGAFDf50anvLPPoz/R7evUNJRS5e95bTM0ayL+p7x6rJbLYBxi6C2b6jx25K5xQ==" },    
```

Comment:
The build/deployment fails if I use a fake storage account name and key. I have to use the ones that exist for the build go through successfully. I am planning to delete the storage account and SQL server after the PR is merged. 

The template creates Data Factory entities in the following order:

1.	Data factory
2.	Azure Storage linked service
3.	Azure SQL Database linked service
4.	Azure Blob dataset
5.	Azure SQL Table dataset
6.	Pipeline

If I give a fake azure storage account name and account key, step #2 fails during build/deploy. Same is the case with the Azure SQL Database linked service. Data Factory service runs the validations in this case.